### PR TITLE
update c2c variables

### PR DESCRIPTION
**Pull request by ddiezrod**
c2c variables files updated so they are now equal to those in perforce

### CODE REVIEW

File: kratos/python/add_c2c_variables_to_python.cpp [hunk 1]
```diff
@@ -40,6 +50,36 @@
 #include "utilities/timer.h"
 
 
+
+#ifdef KRATOS_REGISTER_IN_PYTHON_FLAG_IMPLEMENTATION
```
Comment (RiccardoRossi) [inline kratos/python/add_c2c_variables_to_python.cpp]: ouch, what are u doing here?

Comment (ddiezrod) [reply kratos/python/add_c2c_variables_to_python.cpp]: hmm didnt notice that part. I just copied the files from perfoce... It seems like something that has always been there in the perforce side. Anyway, I have run my c2c tests without that block and everything seems to be fine, so I will just delete that part.

Comment (ddiezrod) [reply kratos/python/add_c2c_variables_to_python.cpp]: Maybe @jrubiogonzalez knows about this?

Comment (roigcarlo) [reply kratos/python/add_c2c_variables_to_python.cpp]: All that define block is no longer necessary (everything until `namespace kratos`)

File: kratos/python/add_c2c_variables_to_python.cpp [hunk 2]
```diff
@@ -112,6 +152,11 @@ void  AddC2CVariablesToPython()
 	KRATOS_REGISTER_IN_PYTHON_VARIABLE(COLD_SHUTS)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(NIYAMA)
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(TEMPERATURE_GRADIENT)
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(INITIAL_TEMPERATURE)
 }
-} // namespace Python.
+}  // namespace Python.
 } // Namespace Kratos
+
+#undef KRATOS_REGISTER_IN_PYTHON_FLAG
```
Comment (roigcarlo) [inline kratos/python/add_c2c_variables_to_python.cpp]: These defines are also not necessary anymore

File: kratos/sources/c2c_variables.cpp [hunk 3]
```diff
@@ -22,6 +22,42 @@
...
+// #include "geometries/prism_3d_6.h"
+// #include "geometries/prism_3d_15.h"
+// #include "geometries/hexahedra_3d_8.h"
+// #include "geometries/hexahedra_3d_20.h"
+// #include "geometries/hexahedra_3d_27.h"
+
+// #include "python/add_c2c_variables_to_python.h"
+
+// #include "includes/convection_diffusion_settings.h"
+// #include "includes/radiation_settings.h"
```
Comment (roigcarlo) [inline kratos/sources/c2c_variables.cpp]: Those includes are also not needed to add comments

**Review by roigcarlo — COMMENTED** [review]
I would suggest to remove the old code in general rather than leave it commented unless is something that will be used in the future as it makes the sources hard to read. Specially the variables ones if you are looking for an specific var

**Comment by ddiezrod** [comment]
@roigcarlo @RiccardoRossi  Ok, I'll do this right and clean everything.

**Review by roigcarlo — CHANGES_REQUESTED** [review]
ThermoMechanical application breaks. I am not sure who is the responsible for that one or if we should move the affected variables elsewhere. @RiccardoRossi ?

The list of needed variables by the Thermomechanical app is:

* IS_SOLIDIFIED
* POSETIVE_DISTANCE
* NAGATIVE_DISTANCE
* PARTICLE_RADIUS
* IS_ESCAPED
* NODE_PROPERTY_ID
* REF_ID

**Comment by pooyan-dadvand** [comment]
Just a cosmetic comment, @ddiezrod why you are changing the new header comments to the old ones?

File: kratos/python/add_c2c_variables_to_python.cpp [hunk 4]
```diff
@@ -1,15 +1,24 @@
-//    |  /           |
```
Comment (pooyan-dadvand) [inline kratos/python/add_c2c_variables_to_python.cpp:1]: This is the new header comment

**Comment by ddiezrod** [comment]
@pooyan-dadvand  for absolutely no reason. Just modified that. @roigcarlo In my opinion, only variables that are used in c2c should be included here, right? @RiccardoRossi  @jrubiogonzalez

**Comment by roigcarlo** [comment]
@ddiezrod Yep, probably should be somewhere else. But I am not sure @pooyan-dadvand ?

**Comment by jrubiogonzalez** [comment]
I also think that if they are needed in Thermomechanical application, they should be either in Thermomechanical or in core. In the long run we'd like to reduce this list of variables to the very minimal. It showed to be a source of error for other applications, but we are not in a position to remove them completely.

**Comment by pooyan-dadvand** [comment]
I would add them to the deprecated_variables.h and cpp. @KratosMultiphysics/technical-committee ?

**Comment by ddiezrod** [comment]
@KratosMultiphysics/technical-committee If nobody has something against this, I will move the variables mentioned by @roigcarlo  plus IS_GRAVITY_FILLING to deprecated variables

**Comment by jcotela** [comment]
@ddiezrod That's OK for me, 👍

**Comment by ddiezrod** [comment]
Done @roigcarlo , I checked ThermoMechanical application compiles again.

**Review by roigcarlo — APPROVED** [review]
Good to me! :+1:

File: kratos/python/add_deprecated_variables_to_python.cpp [hunk 5]
```diff
@@ -140,6 +140,16 @@ void  AddDeprecatedVariablesToPython()
 
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS( xi_c )
 
+	//for Thermo Mechanical Application
+	KRATOS_REGISTER_IN_PYTHON_VARIABLE(NODE_PROPERTY_ID)
+	KRATOS_REGISTER_IN_PYTHON_VARIABLE(REF_ID)
+	KRATOS_REGISTER_IN_PYTHON_VARIABLE(PARTICLE_RADIUS)
+	KRATOS_REGISTER_IN_PYTHON_VARIABLE(POSETIVE_DISTANCE)
```
Comment (maceligueta) [inline kratos/python/add_deprecated_variables_to_python.cpp:147]: posEtive? Just remarking, I don't really mind.

Comment (roigcarlo) [reply kratos/python/add_deprecated_variables_to_python.cpp:147]: Yep, its this way in the application as well, we can make a different issue for that one.

**Comment by jrubiogonzalez** [comment]
Thanks @ddiezrod

**Pull request merged**